internal/winapi/net.go

```diff
@@ -0,0 +1,3 @@
+package winapi
+
+//sys SetJobCompartmentId(handle windows.Handle, compartmentId uint32) (hr error) = iphlpapi.SetJobCompartmentId
```

Comment:
I don't think naming the return code `hr` is correct here. That will cause HRESULT return value semantics to be used, whereas this function actually appears to return a standard win32 error code. Given that the two types have different definitions of what constitutes an error, this could result in false positives and false negatives.

If you change the error variable name to something like `win32Err`, I think it will generate the correct code.

Reply:
I think I did this because SetCurrentThreadCompartmentId is also defined this way in our repo. Can remedy

Reply:
`SetCurrentThreadCompartmentId` looks wrong as well, but also doesn't appear to be used anywhere.

Reply:
Not here correct. Its used in docker somewhere IIRC